control-plane/rest/openapi-specs/v0_api_spec.yaml

```
@@ -2978,6 +3028,37 @@ components:
       required:
         - spec
         - state
+    Snapshot:
+      description: |-
+        Snapshot Information
+      type: object
+      properties:
+        uuid:
+          $ref: '#/components/schemas/SnapshotId'
+        name:
+          description: Name of the snapshot
+          type: string
+        size:
+          description: Size that the snapshot should be.
+          type: integer
+          format: int64
+          minimum: 0
+        source:
+          $ref: '#/components/schemas/VolumeId'
+        creation_timestamp:
+          description: Timestamp when the point-in-time snapshot is taken on the storage system.
+          type: string
+          format: date-time
+        ready:
+          description: Indicates if a snapshot is ready to use as a `volume_content_source` in a `CreateVolumeRequest`.
```

Until we support such thing at rest, maybe simply:
```suggestion
          description: Indicates if a snapshot is ready to be used as a clone.
```